### ISSUE TITLE
Make a nicer options UI

### ISSUE THREAD

**Issue by ceg9498** [ceg9498/kanji-bingo#1]
**Problem**
The UI for the options section (at the top left) is just using the default HTML elements and hasn't been styled at all, so it doesn't look very nice.

**Solution**
* The options section should be a nice-looking menu bar along the top.
* The buttons and select menu should remain in default HTML tags to maintain accessibility.

**Comment by Maryam-Momoh** [comment]
hello, i am interested in this task, can i be assigned to it

**Comment by ldnminh31** [comment]
Hi, can I take this one? 😁

**Comment by ceg9498** [comment]
Hi - 
@Maryam-Momoh Yes, I will assign you to this issue.
@ldnminh31 You can work on it if you like, but priority will be given to Maryam-Momoh.

Thank you both for your interest!

**Comment by Maryam-Momoh** [comment]
https://user-images.githubusercontent.com/75956011/196044559-b01d1976-11bd-4dfc-9e07-6f78074e093e.mp4


https://user-images.githubusercontent.com/75956011/196044566-467abf89-0a5f-4d38-bd65-e00cd5189c4a.mp4


https://user-images.githubusercontent.com/75956011/196044575-2c7b94e0-4f00-4751-88e5-0c6f64249601.mp4


https://user-images.githubusercontent.com/75956011/196044588-40aea7a0-1e53-40e5-a583-2e34572c1b7e.mp4



which one looks good?

**Comment by ceg9498** [comment]
@Maryam-Momoh I like the 2nd one with the pink underline the best!

**Comment by Maryam-Momoh** [comment]
Great, good for PR?

**Comment by ceg9498** [comment]
Yes - go ahead and submit a PR. I will review it as soon as I'm able.
